_includes/components/charts/chart.html

```diff
@@ -5,11 +5,18 @@
     </div>
 
     {% assign graph_template = 'components/charts/' | append: include.graph_type | append: '.html' %}
-    <div class="plot-container">
-        {% include {{graph_template}} %}
+    <div id="selectionsChart">
+      <div id="chart-canvas">
+        <h3 class="chart-title">{{ meta.graph_title | t }}</h3>
+        <div id="chart" class="plot-container">
+          {% include {{graph_template}} %}
+        </div>
+        <div id="plotLegend"></div>
+      </div>
+      <button id="btnSave" title={{ t.indicator.download_chart_image_title }} class='btn btn-primary btn-download'>
```

Comment:
@brockfanning does `title={{ t.indicator.download_chart_image_title }}` need to be `title="{{ t.indicator.download_chart_image_title }}"` asking since I just tested this [here](https://sdg.mango-solutions.com/html-chart/1-1-1/) and the title seems to be just 'Download' instead of 'Download chart as image' which is what is in the translations file. Tested for armenia locally and same there (just took first word):

![image](https://user-images.githubusercontent.com/43404358/63540612-46d6a280-c514-11e9-95f4-89cccef01aad.png)

Reply:
Yes that's a good point - HTML attributes always need to contained in quotes.